extensions/git/src/commands.ts

```diff
@@ -139,6 +139,7 @@ const ImageMimetypes = [
 export class CommandCenter {
 
 	private disposables: Disposable[];
+	private checkoutItemsUsageList: Array<string>;
```

Comment:
Let's store this in [`workspaceState`](https://github.com/Microsoft/vscode/blob/c7880659857c8d62a70fe001d8539ab987849767/src/vs/vscode.d.ts#L4085) instead.

Reply:
Sure, how do I do that? Didn't see any mentions of `workspaceState` in this file. Could you please point me to a similar usage or a written manual?